Data/src/test/java/org/tribuo/data/columnar/RowProcessorTest.java

```diff
@@ -170,4 +183,127 @@ public void metadataExtractorTest() {
         assertThrows(PropertyException.class, () -> new RowProcessor<>(badExtractors,weightExtractor,response,fixed,Collections.emptySet()));
     }
 
+    @Test
+    public void replaceNewlinesWithSpacesTest() {
+        final Pattern BLANK_LINES = Pattern.compile("(\n[\\s-]*\n)+");
+
+        final Function<CharSequence, CharSequence> newLiner = (CharSequence charSequence) -> {
+            if (charSequence == null || charSequence.length() == 0) {
+                return charSequence;
+            }
+            return BLANK_LINES.splitAsStream(charSequence).collect(Collectors.joining(" ⏎\n\n"));
```

Comment:
What's the character before the `\n\n` here?

Reply:
https://graphemica.com/%E2%8F%8E kinda picked it for fun

Reply:
Ok. Given this is about newline splitting it would probably be better if it wasn't a return symbol in case there's some font glyph replacement on the newlines. So any other sensible emoji is fine, but the return symbol might be confusing.